packages/bot-web-ui/src/components/dashboard/tutorial-tab/common/no-search-result-found/no-search-result.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Icon, Text } from '@deriv/components';
+import { observer } from '@deriv/stores';
+import { localize } from '@deriv/translations';
+import { useDBotStore } from 'Stores/useDBotStore';
+
+const NoSearchResult = observer(() => {
+    const { dashboard } = useDBotStore();
+    const { faq_search_value } = dashboard;
+    return (
+        <div className='dc-tabs__content dc-tabs__content--no-result'>
+            <Icon icon='IcDbotNoSearchResult' size={80} />
+            <Text className='dc-tabs__content--no-result__title' as='h1' weight='bold' line_height='xxs'>
+                {localize('No results found')}
```

Comment:
shouldn't we use the Localize component instead of localize, since its being wrapped with Text component